pipeline/scripts/mac-sign.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const { execSync } = require('child_process');
```

Comment:
Could you include a comment documenting usage and environmental prereqs (eg, the keychain setup it expects has already been done)?